kds/kds.go

```diff
@@ -382,7 +449,22 @@ func kdsOidMapToExtensions(exts map[kdsOID]*pkix.Extension) (*Extensions, error)
 	if err := asn1U8(exts[kdsUcodeSpl], "UcodeSpl", &ucodespl); err != nil {
 		return nil, err
 	}
+
+	switch result.StructVersion {
+	case TCBStructVersion0:
+		fmcspl = 0
```

Comment:
We cannot rely on Go's default values for FMC in the unsupported case as 0 is a valid version number. Perhaps we can introduce a `nil` here?

Reply:
This one just ensures that the `fmcspl` variable is initialized to 0. It is only used if TCB version structure is structVersion1. Otherwise, it's ignored.

Reply:
@larrydewey It was just initializing the unused variables to 0. But I see how it can be confusing.

I've addressed it and marking this conversation as resolved. If anything changes, please let me know.